web/src/message_view_header.ts

```diff
@@ -175,9 +175,10 @@ export function render_title_area(): void {
 // This function checks if "modified_sub" which is the stream whose values
 // have been updated is the same as the stream which is currently
 // narrowed and rerenders if necessary
-export function maybe_rerender_title_area_for_stream(modified_sub: StreamSubscription): void {
+export function maybe_rerender_title_area_for_stream(modified_sub_id: number): void {
```

Comment:
We generally use `stream_id` as paramter name, so would be better to use `modified_stream_id` here.